collection-scripts/gather_network_logs

```diff
@@ -26,6 +26,60 @@ if [ "$NETWORK_TYPE" == "openshiftsdn" ]; then
         oc -n openshift-sdn exec $OVS_POD -- bash -c "cat /var/log/openvswitch/ovsdb-server.log" > ${NETWORK_LOG_PATH}/${OVS_POD}_ovsdb_log &
         PIDS+=($!)
     done
+elif [ "$NETWORK_TYPE" == "kuryr" ]; then
+      CONTROLLER_POD=$(oc -n openshift-kuryr get pods --no-headers -o custom-columns=':metadata.name' -l app=kuryr-controller | tail -1)
+      oc -n openshift-kuryr logs $CONTROLLER_POD > \
+      ${NETWORK_LOG_PATH}/${CONTROLLER_POD}_log &
+      PIDS+=($!)
+
+      CONTROLLER_RESTART_COUNT="${@:-$(oc get po $CONTROLLER_POD -n openshift-kuryr -o jsonpath={.status.containerStatuses[*].restartCount})}"
+      if [ "$CONTROLLER_RESTART_COUNT" -ne "0" ]; then
+          oc -n openshift-kuryr logs $CONTROLLER_POD --previous > ${NETWORK_LOG_PATH}/${CONTROLLER_POD}_previous_log &
+          PIDS+=($!)
+      fi
+
+      NODES="${@:-$(oc get nodes --no-headers -o custom-columns=':metadata.name')}"
```

Comment:
We use this concept in several places; should we try and functionalize it? 
It seems like we're repeating bash code here (where it's possible we will get out of alignment in the various places where we want to restrict collection only to specific nodes). Should we look at handling this differently, more genericly?

Reply:
@dcbw and @knobunc your teams may want to look at what is being done here; just to make sure we're not possibly introducing a bug/feature regression.

Reply:
@sferich888 could you take a look at PR145? We have been working in the same space.